crucible-llvm/src/Lang/Crucible/LLVM/Intrinsics/Common.hs

```diff
@@ -83,7 +83,7 @@ import           Lang.Crucible.LLVM.Translation.Types
 
 -- | This type represents an implementation of an LLVM intrinsic function in
 -- Crucible.
-data LLVMOverride p sym args ret =
+data LLVMOverride p sym ext args ret =
```

Comment:
Worth leaving a comment here about why `LLVMOverride` is parametric in `ext`, as it's a bit surprising if you aren't aware of the use cases.